src/parse/mark.js

```diff
@@ -43,7 +43,6 @@ parseMark.schema = {
             "mark": {"type": "string"},
             "transform": {"$ref": "#/defs/transform"}
           },
-          "oneOf":[{"required": ["data"]}, {"required": ["mark"]}],
```

Comment:
Why was this removed? If a mark's `from` property is specified, it should contains either a `data` or `mark` property.